modules/thunder_liveblog/thunder_liveblog.info.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Thunder Liveblog'
```

Comment:
Add a "Liveblog" paragraph into thunder_liveblog to make it possible to integrate a liveblog into an article